app/build.gradle

```diff
@@ -26,7 +26,21 @@ def tryReadConfig(filename) {
     return config
 }
 
+repositories {
+    maven {
+        url 'https://leiainc.jfrog.io/leiainc/gradle-release'
+    }
+}
+
 android {
+    signingConfigs {
+        debug {
+            storeFile file('C:\\Users\\mail\\Documents\\android-debug-key-store.jks')
+            storePassword 'jakedowns'
+            keyAlias 'jakedownskeystore'
+            keyPassword 'jakedowns'
+        }
+    }
```

Comment:
What specifically is this key for? Does Leia have its own app store? Should I be responsible for publishing to that as well?

(there's no automation around publishing ATM, I just build an aab and yeet it into the play store)

Reply:
oh, heh yeah, i just use it when i do a signed release. i never pushed one to the play store, but it seemed like it was necessary for other people to be able to side-load the apk

Reply:
i will submit a new PR with this removed